pkg/cli/admin/upgrade/recommend/examples/README.md

```
@@ -1,20 +1,30 @@
+* `TESTCASE.output`: expected output of `oc adm upgrade recommend`.
+* `TESTCASE.show-outdated-releases-output`: expected output of `oc adm upgrade recommend --show-outdated-releases`.
+* `TESTCASE.version-<VERSION>-output`: expected output of `oc adm upgrade recommend --to <VERSION>`.
 
 The `TestExamples` test in [`examples_test.go`](../examples_test.go) file above validates all examples.
-When the testcase is executed with a non-empty `UPDATE` environmental variable, it will update the `TESTCASE.out` fixture:
 
+**TESTING**
+---
+* When the testcase is executed with a non-empty `UPDATE` environmental variable, it will update the `TESTCASE.out` fixture:
 ```console
 $ UPDATE=yes go test -v ./pkg/cli/admin/upgrade/recommend/...
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)